test/webserv/integration/test_connection.py

```
@@ -0,0 +1,151 @@
+            assert False
+        else:
+            # serverが200 OKを返したことを確認
+            assert_response(
+                response, response_header_get_root_200_keep + root_index_file
+            )
+
+        # 再度responseを読み込み
+        next_response = receive_with_timeout(con.sock)
+        if next_response is None:
+            # timeout秒経過していたらNG
+            assert False
```

Ruff

Do not `assert False` (`python -O` removes these calls), raise `AssertionError()`

Replace `assert False`

(B011)